randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -640,7 +655,7 @@ Extra - room_id: [17]
           Can Use Power Bombs
           Any of the following:
               Gravity Suit and Space Jump
-              # Use frozen Zebesian to escape
+              # Use frozen enemies to escape: https://youtu.be/4Msq5vtRTzw
               Hi-Jump and Stand On Frozen Enemies (Intermediate) and Wall Jump (Intermediate) and Can Freeze Enemies
```

Comment:
should this really be intermediate? IMO the fact that you can fail this and be stuck should make this a higher bump

Reply:
they reform, but still agreed.